src/modules/mavlink/mavlink_main.cpp

```diff
@@ -1635,7 +1635,7 @@ Mavlink::configure_streams_to_default(const char *configure_single_stream)
 		configure_stream_local("SERVO_OUTPUT_RAW_0", 1.0f);
 		configure_stream_local("SYS_STATUS", 1.0f);
 		configure_stream_local("UTM_GLOBAL_POSITION", 0.5f);
-		configure_stream_local("VFR_HUD", 4.0f);
+		configure_stream_local("VFR_HUD", 15.0f);
```

Comment:
@dagar remember our conversation? So which one is really used?

Reply:
In QGC, once ATTITUDE_QUATERNION is received, ATTITUDE is ignored.

Reply:
Same in MAVSDK.